src/v/features/feature_table.h

```diff
@@ -239,6 +240,12 @@ constexpr static std::array feature_schema{
     feature::rpc_transport_unknown_errc,
     feature_spec::available_policy::always,
     feature_spec::prepare_policy::always},
+  feature_spec{
+    cluster::cluster_version{10},
+    "cloud_log_size_in_manifest",
```

Comment:
Also by the time we release, manifest encoding will have changed totally to binary, so particular logic for this attribute won't be needed -- let's create one big feature flag for "cloud_storage_format_v2" or something like that (v2 referring to the manifest version, which is currently at v1)

Reply:
Went for cloud_storage_manifest_format_v2.